src/demo/player.rs

```diff
@@ -25,15 +27,22 @@ pub(super) fn plugin(app: &mut App) {
     );
 }
 
-#[derive(Event, Debug)]
-pub struct SpawnPlayer;
-
 #[derive(Component, Debug, Clone, Copy, PartialEq, Eq, Default, Reflect)]
 #[reflect(Component)]
 pub struct Player;
 
+/// Command to spawn the player character.
+/// We can add fields to this struct if we need data for spawning.
```

Comment:
```suggestion
/// A command to spawn the player character.
```
I don't like the second line here personally. There are a lot of things the user _could_ do to expand on the template, and we're not going to list them all. At the very least this doesn't make sense as a doc comment.

Reply:
We do however mention things that have a very high probability of coming up and are low-hanging expansions.
Moved the comment into the struct: [`d2c59b6` (#242)](https://github.com/TheBevyFlock/bevy_quickstart/pull/242/commits/d2c59b6ef9162e66971ade4188e99d4246ee75b0)